circle.yml

```diff
@@ -58,6 +58,14 @@ commands:
         working_directory: ~/build
         command: ctest -R ${TESTS_FILTER:-'.*'} -j4 --schedule-random --output-on-failure
 
+  benchmark:
```

Comment:
This won't run with Release mode?

Reply:
Not needed.

Reply:
In that case can you add a first commit which adds benchmarking on ci?

Reply:
Never mind, this is within the asan case.

Reply:
I updated description.

Using CI for benchmarking is actually much more complicated. To be discussed some other time.